README.md

```diff
@@ -0,0 +1,124 @@
+## Rio-tiler
+
+Rasterio plugin to serve mercator tiles from satellite imagery hosted on Amazon S3 buckets.
```

Comment:
@samsammurphy We can maybe also update this line with
`Rasterio plugin to create mercator tiles image from raster sources.` ?

Reply:
Yep, I agree, something like that would be better.